File: Juan/P1/S3/Napakalaki/pkg/Napakalaki-0.0.1/lib/Prize.rb
#encoding: utf-8
# author: Juane
#Versión 1.0
require 'TreasureKind.rb'
class Prize    
    
    def initialize(treasures,level)
        @treasures = treasures
        @levels = level
    end
 
    #Método de consulta primera forma
=begin
def treasures
    @treasures
  end
  
  def level
    @levels
  end
=end
  
  
    #Método toString
 
    def to_s
        "Treasures =  #{@treasures} Niveles ganados =   #{@levels}"
    end
  
    #Método de escritura y lectura segunda forma
    attr_accessor :treasures
    attr_accessor :levels
  
    #Método solo lectura
    # attr_reader	
    #Método solo escritura
    # attr_writer
end

File: Juan/P1/S1/Napakalaki/src/napakalaki/Prize.java
package napakalaki;

/**
 *
 * @author juane
 * @version 1.0
 *
 * Clase Prize para el buen rollo
 */
public class Prize {

    private int treasures;
    private int level;

    /**
     * Constructor
     */
    Prize(int treasures, int level) {
        this.treasures = treasures;
        this.level = level;
    }

    /**
     * Consultores
     */
    /**
     * Método getTreasures()
     *
     * @return treasures de tipo int
     */
    public int getTreasures() {
        return this.treasures;
    }

    /**
     * Método getLevel()
     *
     * @return level de tipo int
     */
    public int getLevel() {
        return this.level;
    }

    @Override
    public String toString() {
        return "Treasures = " + Integer.toString(treasures) 
                + ", levels = " + Integer.toString(level) ;
    }
    
}
